offchain/eth-input-reader/src/machine/driver.rs

```diff
@@ -5,7 +5,8 @@ use super::Context;
 
```

Comment:
I'd reconsider the name of `MachineDriver`. In retrospect, it's not amazing. And since there's no longer it's dual (`BlockchainDriver`), I'd try to think about something better.